dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-rules.ts

```diff
@@ -67,9 +67,10 @@ export function useRules(
     if (ruleLoading.value) return
     ruleLoading.value = true
     const result = await getRuleFormCreateJson(ruleId)
-    const items = JSON.parse(result).map((item: IResponseJsonItem) =>
-      formatResponseJson(item)
-    )
+    const items = JSON.parse(result).map((item: IResponseJsonItem) => {
+      item.props.placeholder = t('project.node.' + item.props.placeholder.split(' ').join('_').split(',').join('').toLowerCase())
+      return formatResponseJson(item)
+    })
```

Comment:
It's better to handle the placeholder format in the formatResponseJson function as well.